scdl/scdl.py

```diff
@@ -298,6 +315,9 @@ def download(user, dl_type, name):
                 download_playlist(item)
             elif dl_type == 'playlists-liked':
                 parse_url(item['playlist']['uri'])
+            elif dl_type == 'commented':
+                item=get_track_info(item['track_id'])
+                download_track(item)
```

Comment:
item here is actually a collection. You cannot directly pass it to download_track.
Did you tried to run this code ?